EssentialFeed/EssentialFeed/Comments/Presentation/CommentsPresenter.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  CommentsPresenter.swift
+//  EssentialFeed
+//
+//  Created by Robert Dates on 1/23/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+
+public final class CommentsPresenter {
```

Comment:
Remove extra line break

```suggestion
import Foundation

public final class CommentsPresenter {
```

Reply:
done